exercises/bowling/example.rb

```diff
@@ -20,7 +20,7 @@ def roll(pins)
   end
 
   def validate(pins)
-    raise 'Invalid number of pins' unless (RULES[:MIN]..RULES[:MAX]).cover?(pins)
+    raise 'Invalid number of pins' unless (PINS[:MIN]..PINS[:MAX]).cover?(pins)
```

Comment:
The kata is about scoring bowling, not worrying if your test inputs are valid. So I would argue against adding test cases for it. 
It *is* something that could be discussed during solution reviews.